qstash/workflows/platforms/nuxt.mdx

```diff
@@ -2,61 +2,134 @@
+    // see step 4 for details
+    baseUrl: process.env.NODE_ENV === "development" ? "https://<YOUR_NGROK_OR_TUNNEL_URL>/" : undefined,
+  }
 )
 ```
 
-See [the documentation of the `serve` method for more details](/qstash/workflows/basics/serve).
+## Step 4: Run the Workflow Endpoint
 
-# 5. Run Locally
+For QStash, a tool that's used to run workflows under the hood, you need to provide a live URL for your workflow endpoint. Here's how to [set up your workflow endpoint for local development](qstash/workflows/howto/local-development).
 
-Finally, we can run the project locally. Simply run:
+In production, a workflows `baseUrl` is automatically set and can be omitted.
 
-```
+Once you have a live URL for your local workflow endpoint, trigger your workflow by first starting your Nuxt app:
+
+```bash Terminal
 npm run dev
 ```
 
-To trigger the endpoint, you can run:
+and then making a POST request to your workflow endpoint. For each workflow run, a unique workflow run ID is returned:
 
+```bash Terminal
+curl -X POST https://<YOUR_NGROK_OR_TUNNEL_URL>/api/workflow
+
+# result: {"workflowRunId":"wfr_xxxxxx"}
 ```
-curl -X POST <DEV-URL>/workflow-endpoint -d '{"text": "hello world!"}'
-```
 
-Note that if you set environment variables `QSTASH_CURRENT_SIGNING_KEY` and `QSTASH_NEXT_SIGNING_KEY`; your curl request with fail since the endpoint only accepts requests coming from QStash. [See "Secure Endpoints" for how you can call these endpoints](/qstash/workflows/howto/security).
+<Frame>
+  {/* also works for nuxt */}
+  <img src="/img/qstash-workflow/nextjs_local_request.png" />
+</Frame>
+
+Use this ID to track the workflow run and see its status in your QStash workflow dashboard. All steps are listed with their statuses, headers, and body for a detailed overview of your workflow from start to finish. Click on a step to see its detailed logs.
+
+<Frame>
+  <img src="/img/qstash-workflow/dashboard.png" />
+</Frame>
+
+## Step 5: Deploying to Production
+
+When deploying your Nuxt.js application with Upstash Workflows to production, there are a few key points to keep in mind:
+
+1. **Environment Variables**: Make sure that all necessary environment variables are set in your platforms project settings. For example, your `QSTASH_TOKEN` and any other configuration variables your workflow might need.
+
+2. **Remove Local Development Settings**: In your production code, you can remove or conditionally exclude any local development settings. For example, the `baseUrl` option in the `serve` function can be omitted in production:
+
+   ```typescript app/api/workflow/route.ts
+   export const POST = serve(
+     async (context) => {
+       // Your workflow steps
+     },
+     {
+       // Conditionally set in development, but not in production
+       // as the baseUrl is automatically set in production
+       baseUrl:
+         process.env.NODE_ENV === "development"
+           ? "https://<YOUR_NGROK_OR_TUNNEL_URL>/"
+           : undefined,
```

Comment:
baseUrl is not set in production.

I think it's better if we use UPSTASH_WORKFLOW_URL in development and not set it at all in production @joschan21 

We can also remove "Remove Local Development Settings" from Step 5 if we use UPSTASH_WORKFLOW_URL. We can just say they can omit UPSTASH_WORKFLOW_URL in "Environment Variables"

Same goes for other platforms, cf & next

Reply:
got it and agree, addressed in following commit